oauth/v1/generated.proto

```diff
@@ -142,6 +145,10 @@ message OAuthClient {
   // AccessTokenMaxAgeSeconds overrides the default access token max age for tokens granted to this client.
   // 0 means no expiration.
   optional int32 accessTokenMaxAgeSeconds = 8;
+
+  // AccessTokenTimeoutSeconds overrides the default token timeout for tokens
+  // granted to this client. 0 means no timeout.
+  optional int32 accessTokenTimeoutSeconds = 9;
```

Comment:
What is the difference between maxAge and TimeoutSeconds?

Reply:
If this is about inactivity, this should be `AccessTokenInactivityTimeoutSeconds`

Reply:
ok